moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -87,15 +86,27 @@ const ComponentForm = ({
   const areFormErrors = Object.keys(errors).length > 0;
 
   // Get and format component and subcomponent options
-  const {
-    data: optionsData,
-    loading: areOptionsLoading,
```

Comment:
This loading state was only being used to prevent a MUI autocomplete error about it needing options. The hooks that provide the options feed an empty array now so this check isn't needed.